_health-care/_js/_components/mothers_maiden_name.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import _ from 'lodash';
+
+class MothersMaidenName extends React.Component {
+  constructor() {
+    super();
+    this.state = {name: ''};
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleNameChange(event) {
+    this.setState({name: event.target.value});
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="usa-input-grid usa-input-grid-large">
+          <label htmlFor="veteran_mothers_maiden_name">Mother’s Maiden Name</label>
+          <input type="text" name="veteran[mothers_maiden_name]"
```

Comment:
Yes...I was thinking the object state structure would look something like

`App.veteran = { mothers_maiden_name: {} }`

as the initial state of the top level App component.

And then we'd do in App's `render()` method:
  `<MothersMaidenName state=this.veteran.mothers_maiden_name>`